README.md

```diff
@@ -1364,4 +1364,7 @@ My name is [Kostya](https://github.com/hattorihanz0/) and I am so excited to be
 
 ---
 # Hi mates :D
-![Cat](gif/fat-kottan.gif)
\ No newline at end of file
+![Cat](gif/fat-kottan.gif)
+
+
+# HELLO!!!
```

Comment:
Good job!
But if you can add a bit more info about yourself it will be great?
Thanks!